legacy/app/data/common/post-edit-create/location.directive.js

```diff
@@ -187,11 +187,15 @@ function PostLocationDirective($document, $http, L, Geocoding, Maps, _, Notify,
             element[0].querySelector('#searchbar-results').classList.remove('active');
         }
 
+        function test1() {
+            console.log('chege test');
+        }
+
         function handleActiveSearch(event) {
-            var del = event.keyCode === 8 || event.keyCode === 46;
-            var letter = event.keyCode > 47 && event.keyCode < 58;
-            var number = event.keyCode > 64 && event.keyCode < 91;
-            if (del || letter || number) {
+            // var del = event.keyCode === 8 || event.keyCode === 46;
```

Comment:
@chegejohn159 You can also clean up the code: remove the comments and remove the test1 function you added earlier. After that, let's see if @tuxpiper is ok with the latest changes.

Also I see some other files apart from the ones you edited. The entire .idea folder. You may want to remove those too.

Reply:
Done, removed .idea, removed the comments and the extra function

Reply:
It is not complete @chegejohn159 
Restore the package-lock files too to be on the safe side. The only file you made a change to is the location directive. So when you click on the "Files changed" tab here on this PR, the location directive file should be the only file shown as changed there.
That is what I meant by restoring all other changes.
Once you do this, PR will be approved.